code/game/objects/items/stacks/stack.dm

```diff
@@ -75,7 +75,7 @@
 /obj/item/stack/examine(mob/user)
 	if(..(user, 1))
 		if(!uses_charge)
-			to_chat(user, "There [src.amount == 1 ? "is" : "are"] [src.amount] [src.singular_name]\s in the stack.")
+			to_chat(user, "There [src.amount == 1 ? "is" : "are"] [src.amount] [src.singular_name]\s in the stack.") //will say that there are multiple kni!F!es, but who cares?
```

Comment:
I care. Overwrite that proc in `/obj/item/stack/throwing_knife` or account for `throwing_knife` in this one.

Reply:
Overwrote, all good I think